add-books.js

```diff
@@ -1,51 +1,36 @@
-const title = document.querySelector('#title');
-const author = document.querySelector('#author');
-const add = document.querySelector('.submit');
-const allBooks = document.querySelector('#all-books');
-
-const bookStore = JSON.parse(localStorage.getItem('books')) || [];
-
-function showBooks() {
-  const displayBooks = bookStore.map((book, index) => `
-    <article class="books">
-      <p>${book.title}</p>
-      <p>${book.author}</p>
-      <button class="remove" onclick="removeBook(${index})">Remove</button>
-      <hr />
-    </article>
-`);
-  allBooks.innerHTML = displayBooks.join('');
-}
-
-function addBooks() {
-  if (title.value !== '' && author.value !== '') {
-    bookStore.push({ title: title.value, author: author.value });
-  }
-}
-
-function saveBooks() {
-  localStorage.setItem('books', JSON.stringify(bookStore));
-}
-
-function clearInput() {
-  title.value = '';
-  author.value = '';
-}
-
-function removeBook(index) { // eslint-disable-line no-unused-vars
-  bookStore.splice(index, 1);
-  saveBooks();
-  showBooks();
-}
-
-add.addEventListener('click', (e) => {
-  e.preventDefault();
-  addBooks();
-  saveBooks();
-  showBooks();
-  clearInput();
-});
-
-window.addEventListener('DOMContentLoaded', () => {
-  showBooks();
-});
\ No newline at end of file
+/* eslint-disable import/first */
+/* eslint-disable-line no-unused-vars */
+
+import { BookCollection } from './modules/methods.js';
+
+// Book collection
+const bookCollection = new BookCollection();
+
+bookCollection.addBook();
+
+// Displays all the initiated variables
+import {
+  listBook, newBook, contact, section1, section2, section3, logo, separator,
+} from './modules/variables.js';
+
+// Displays addBooks at default
+section1.style.display = 'none';
+section2.style.display = '';
+section3.style.display = 'none';
+separator.style.display = 'none';
+logo.innerHTML = 'Awesome Books';
+
+import {
+  handleListBookClick,
+  handleNewBookClick,
+  handleContactClick,
+} from './modules/events.js';
+
+listBook.addEventListener('click', handleListBookClick);
+newBook.addEventListener('click', handleNewBookClick);
+contact.addEventListener('click', handleContactClick);
+
+// Add date and time to page
+import dateTimeModule from './modules/time.js';
+
+dateTimeModule();
```

Comment:
Kindly rename this file from `add-book.js` to `index.js` as the instruction given was that `Create the entry point for your JavaScript code called index.js`